odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewStats/OverviewDatasetStats/OverviewDatasetStatsStyles.ts

```diff
@@ -1,27 +1,13 @@
-import { Theme } from '@mui/material';
+import { Grid, Typography } from '@mui/material';
 
-import { WithStyles } from '@mui/styles';
-import createStyles from '@mui/styles/createStyles';
+import { styled } from '@mui/material/styles';
 
-export const styles = (theme: Theme) =>
-  createStyles({
-    statsItem: {
-      display: 'flex',
-      alignItems: 'center',
-    },
-    statIcon: {
-      fontSize: theme.typography.h5.fontSize,
-      marginRight: theme.spacing(1),
-    },
-    statLabel: {
-      color: '#B3BAC5',
-      textTransform: 'uppercase',
-    },
-    statValue: {},
-    typeLabel: {
-      marginLeft: 0,
-      marginBottom: theme.spacing(1.25),
-    },
-  });
+export const StatsItem = styled(Grid)(() => ({
+  display: 'flex',
```

Comment:
If you use container prop of the Grid it will be displayed flex by default, so you can pass alignItems prop to the Grid and StatsItem component won't be needed.